model/Configuration.schema

```diff
@@ -47,9 +47,9 @@
             "properties" : {
                 "usage": {"type" : "number", "minimum": 0.0},
                 "detection_delay": {"type" : "number", "minimum": 0.0},
-                "testing_delay": {"type" : "number", "minimum": 0.0}
+                "usage_by_household": {"type" : "boolean"}
```

Comment:
I am not familiar with this change - just to ensure, is this expected `testing_delay` is dropped and `usage_by_household` is added? @tozanski

Reply:
yes this is correct, `testing_delay` is not yet used - it was an experiment on another branch and somehow made it into the example JSON